src/components/Nav/styles.js

```diff
@@ -0,0 +1,52 @@
+import styled, {css} from 'styled-components/macro';
+
+export const HeaderNav = styled.nav`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    box-sizing: border-box;
```

Comment:
Imho you want this on all elements to save you some headache in the future, think about adding it to a GlobalStyle